src/functionalTest/resources/scenarios/RIA-3308-home-office-notifications-for-request_respondent_evidence.json

```diff
@@ -0,0 +1,35 @@
+{
+  "description": "RIA-3308 sends home office notifications for request respondent evidence",
+  "enabled": "{$featureFlag.isHomeOfficeIntegrationEnabled}",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "CaseOfficer",
+    "input": {
+      "eventId": "requestRespondentEvidence",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeReferenceNumber": "1111-2222-3333-4444",
+          "appealSubmissionDate": "2020-09-22",
+          "sendDirectionExplanation": "A notice of appeal has been lodged against this decision.\n\nYou must now upload all documents to the Tribunal. The Tribunal will make them accessible to the other party. You have 14 days to supply the documents.\n\nYou must include:\n- the notice of decision\n- any other document provided to the appellant giving reasons for that decision\n- any statements of evidence\n- the application form\n- any record of interview with the appellant in relation to the decision being appealed\n- any other unpublished documents on which you rely\n- the notice of any other appealable decision made in relation to the appellant",
+          "sendDirectionParties": "respondent",
+          "sendDirectionDateDue": "{$TODAY+14}",
+          "isHomeOfficeIntegrationEnabled": "Yes"
```

Comment:
The test will run if featureFlag is enabled. Do we have to set the variable again?

Reply:
We use the the existing featureFlag for vaildation now. This is added here to depict that at this stage this flag should be set. There will be a dedicated flag added for notifications later.

Reply:
It's been removed.